frontend/src/components/DarkModeButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import moon from '../images/darkmode.png';
+import sun from '../images/lightmode.png';
+
+import common_styles from '../styles/common.module.css';
+
+/**
+ * DarkMode Button
+ * @prop darkModeEnabled - boolean to determine which image to display (true for "sun"; false for "moon")
+ */
+
+function DarkModeButton({ darkModeEnabled = false }) {
+  return (
+    <span>
+      <img
+        src={darkModeEnabled ? sun : moon}
```

Comment:
We already have the react-icons library and we're mostly using icons from font-awesome.

The FaSun and FaMoon/FaRegMoon might be a good fit, and then we can avoid the extra png's